getpaid/utils.py

```diff
@@ -81,3 +82,16 @@ def qs_to_ordered_params(query_string):
             field = field.decode('utf8')
         params[field] = value
     return params
+
+
+def get_domain(request=None):
+    if request and request.META.get('HTTP_HOST'):
```

Comment:
Question is can we use here a `SITE_URL` or better will be to use some getpaid specific settings constant like `GETPAID_SITE_URL`, and if it's `None` then use `SITE_URL`